pkg/reconciler/service/resources/configuration_test.go

```diff
@@ -127,3 +129,14 @@ func TestInlineConfigurationSpec(t *testing.T) {
 		t.Errorf("expected %q labels got %q", want, got)
 	}
 }
+
+func TestConfigurationHasNoKubectlAnnotation(t *testing.T) {
+	s := createServiceWithKubectlAnnotation()
+	c, err := makeConfiguration(s)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
```

Comment:
```suggestion
		t.Fatalf("Unexpected error: %v", err)
```
You need to terminate test here, or it'll panic below.